src/main/scala/no/ndla/draftapi/service/StateTransitionRules.scala

```diff
@@ -103,12 +103,16 @@ trait StateTransitionRules {
        DRAFT                         -> DRAFT,
        DRAFT                         -> PROPOSAL,
        DRAFT                         -> ARCHIVED                       require PublishRoles illegalStatuses Set(PUBLISHED),
+       DRAFT                         -> AWAITING_ARCHIVING,
       (DRAFT                         -> PUBLISHED)                     keepStates Set(IMPORTED) require DirectPublishRoles withSideEffect publishWithSoftValidation,
        ARCHIVED                      -> ARCHIVED,
        ARCHIVED                      -> DRAFT,
+       AWAITING_ARCHIVING            -> AWAITING_ARCHIVING,
+       AWAITING_ARCHIVING            -> ARCHIVED                       require PublishRoles illegalStatuses Set(PUBLISHED),
```

Comment:
Når eg tenker meg om må vi nesten ha med ` withSideEffect  unpublishArticle` her. For då kan artiklene avpubliseres direkte fra denne statusen.
```suggestion
       AWAITING_ARCHIVING            -> ARCHIVED                       require PublishRoles illegalStatuses Set(PUBLISHED) withSideEffect  unpublishArticle,
```